geonode/base/api/tests.py

```diff
@@ -641,7 +641,7 @@ def test_write_resources(self):
             self.assertEqual('Foo Abstract', response.data['resource']['abstract'], response.data['resource']['abstract'])
             self.assertEqual('Foo Attribution', response.data['resource']['attribution'], response.data['resource']['attribution'])
             self.assertEqual('321-12345-987654321', response.data['resource']['doi'], response.data['resource']['doi'])
-            self.assertEqual(False, response.data['resource']['is_published'], response.data['resource']['is_published'])
+            self.assertEqual(True, response.data['resource']['is_published'], response.data['resource']['is_published'])
```

Comment:
@mattiagiupponi if bobby is the owner he should be able to publish the resource, unless the advanced workflow is activated. So this asswertion should match `False`, isn't it?

Reply:
As decided in the call, we set the `is_published` and `is_approved` to read_only params. This means that they cannot be updated